tethysapp/water_data_explorer/sites.py

```diff
@@ -403,134 +402,134 @@ def _getSiteInfoHelper(object_siteInfo,object_methods):
             if str(sitePorperty_Info['@name']) == 'Country':
                 return_obj['country'] = str(sitePorperty_Info['#text'])
                 # print(return_obj['country'])
-    except Exception as e:
+    except Exception:
         # print(e)
         return_obj['country'] = "No Data was Provided"
     try:
         # return_obj['siteName'] = object_siteInfo['siteName']
         siteName = object_siteInfo['siteName'].encode("utf-8")
         # return_object['siteName'] = siteName
         return_obj['siteName'] = siteName.decode("utf-8")
-    except KeyError as ke:
+    except Exception:
```

Comment:
Why were all of these changed to ``Exception`` instead of ``KeyError``?

Reply:
By mistake....  fixed.